### DISCUSSION TITLE
Open file from terminal to code

### DISCUSSION THREAD

**Discussion by omar2205** [coder/code-server#5404] [Q&A]
How do I open a file in the terminal with code-server?

When I try `code-server a` I get
`/usr/lib/code-server/lib/vscode/bin/remote-cli/code-server: 12: /usr/lib/code-server/lib/vscode/node: not found`

**Comment by jsjoeio** [comment]
Are you running `code-server a` inside code-server?

**Reply by omar2205** [reply]
Yes

**Reply by omar2205** [reply]
In the integrated terminal, I want to open file.txt in code

**Reply by jsjoeio** [reply]
Oh we had a bug that will be fixed in the next release. See here: https://github.com/coder/code-server/pull/5360

**Reply by omar2205** [reply]
alright, thanks!
